### PR TITLE
chore: formatting release post

### LINKED ISSUE #6041: Run Prettier as part of the blog post script
Now we have to run Prettier on blog posts? Please do it as part of the script that generates them!

_Originally posted by @targos in https://github.com/nodejs/nodejs.org/issues/6040#issuecomment-1777304015_

See https://github.com/nodejs/nodejs.org/blob/main/scripts/release-post/index.mjs - add a exec call to run prettier in some way.

You can validate this works by re-running the 21.1.0 release from which this  issue came

### PR DESCRIPTION
**Pull request by huynl-96**
## Description

Run prettier before writing release post to file.

## Validation

WITH vs WITHOUT prettier format version respectively using diff changes comparison

https://github.com/nodejs/nodejs.org/assets/15758406/f7ce9bb3-75be-4f46-92b0-b20afc054e0c

image

## Related Issues

Resolves #6041 

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [ ] I have run `npx turbo test` to check if all tests are passing.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: scripts/release-post/index.mjs [hunk 1]
```diff
@@ -25,6 +25,7 @@ import { writeFile } from 'node:fs/promises';
 import { resolve } from 'node:path';
 
 import handlebars from 'handlebars';
+import { format } from 'prettier';
```
Comment (ovflowd) [inline scripts/release-post/index.mjs:28]: Does this read our prettier settings by default? `.prettierrc.json`

Comment (huynl-96) [reply scripts/release-post/index.mjs:28]: Not yet, I'm currently using default options. 
Will import `.prettierrc.json` configs for custom options. Thanks 🙇🏻‍♂️

File: scripts/release-post/index.mjs [hunk 2]
```diff
@@ -208,6 +212,16 @@ const renderPost = results => {
   return { content: template(templateParameters), ...results };
 };
 
+const formatPost = results => {
+  return new Promise((resolve, reject) => {
+    format(results.content, { ...prettierConfig, parser: 'markdown' })
+      .then(formattedContent =>
```
Comment (ovflowd) [inline scripts/release-post/index.mjs]: Can we call this variable just content? I think that's fine haha

**Review by ovflowd — APPROVED** [review]
Awesome and quick work! Thank you so much for your first contribution :shipit:

File: scripts/release-post/index.mjs (hunk 2 above)
Comment (ovflowd) [reply scripts/release-post/index.mjs]: Let me know once you've updated that. But we can also merge it as it is.

Comment (huynl-96) [reply scripts/release-post/index.mjs]: Updated in 8b5c81c 🤝

**Review by mikeesto — APPROVED** [review]
LGTM, thanks for your first contribution

**Review by aymen94 — APPROVED** [review]

**Pull request merged**